cmd/helm/deployment.go

```diff
@@ -50,7 +56,46 @@ func deploymentCommands() cli.Command {
 				Name:      "list",
 				Usage:     "list all deployments, or filter by an optional pattern",
 				ArgsUsage: "PATTERN",
+				Action:    func(c *cli.Context) { run(c, list) },
 			},
 		},
 	}
 }
+
+func listCmd() cli.Command {
```

Comment:
These functions names might need to be name spaced because of `repo list`, `cred list`, `chart list`, ...